CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # How to contribute
 
-In this repo we follow the conventions defined in Vincent Driessen's **[git branching model](http://nvie.com/posts/a-successful-git-branching-model/)**. See **[git-flow](https://github.com/nvie/gitflow)** if you want some git extensions to help you with the mentioned model.
+In this repo, Feature Branch Workflow is the way we work with GIT. Basically this is based on creating branches that start from the master called feature branch. These branches have a single purpose, that is, they are created to add a specific functionality, to correct a bug, to delete deprecated functionality etc ... They will always be used for something specific. Once you complete what is being done in the feature branch, this functionality must be joined to master again and thus repeat the cycle during development.
 
 We **try** to use [Semantic Versioning](http://semver.org) and **Scrum** in our software development processes _as much as possible_. The [issues](https://github.com/iporaitech/pwr2-docker/issues) are grouped into [milestones](https://github.com/iporaitech/pwr2-docker/milestones) and most of the times will be labeled as: **Feature**, **Bug**, **Task**.  
```

Comment:
Finally @rsjavier14, see if there are any typos and update the above description in case something missing or should be removed.